src/datasources/asset/AssetDataSource.ts

```diff
@@ -82,4 +84,11 @@ export class AssetDataSource extends DataSourceBase<AssetQuery, AssetDataSourceO
   getListAssetsSource(): ListAssetsDataSource {
     return this.listAssetsDataSource;
   }
+
+  async metricFindQuery(query: AssetQuery): Promise<MetricFindValue[]> {
+    console.log(query);
+    const assetFilter = query as ListAssetsQuery;
```

Comment:
`const assetFilter = assetFilter?.filter ?? '';`

this should probably be enough as well as we use only that property.